src/auth/OAuth.ts

```diff
@@ -278,14 +307,14 @@ export class OAuth extends BaseAuthAPI {
    */
   async clientCredentialsGrant(
     bodyParameters: ClientCredentialsGrantRequest,
-    initOverrides?: InitOverride
+    options: { initOverrides?: InitOverride } = {}
```

Comment:
I'd like to change the signature of `initOverrides` from a positional argument to a config object in all the client's methods. So we can add other optional arguments to some of the methods

Reply:
Are you saying to also patch the management side?